js/src/csp/components/CSPVisualizer.vue

```diff
@@ -134,4 +167,31 @@ export default class CSPGraphInteractor extends Vue {
   text.domain {
     font-size: 12px;
   }
+
+  .btn-text {
```

Comment:
Do we need this css? This seems different to the css for the SearchVisualizer dropdown. A file with all the common css could be a good idea.